api/bases/ovs.openstack.org_ovs.yaml

```diff
@@ -56,6 +56,11 @@ spec:
                 - ovn-bridge
                 - system-id
                 type: object
+              networkAttachment:
```

Comment:
would there be a uses case to attach the pods to multiple networks and have another param to specify the one which references the network from which the interface IP should be used for the OvnEncapIP?

Reply:
sorry for being late into this but I think that there is such use case as this pod should be connected usually to "internalapi" and to "tenant" networks for tunnels. And that "tenant" network should be used as the one from which OvnEncapIP is taken. I will propose update for that to the ovs-operator

Reply:
Actually no, it needs to connect to ovn central pods but it can use "pods" network probably, right?